src/content/community/releases/_index.en.md

```diff
@@ -55,6 +55,8 @@ weight = 40
   potential attack surface.
 * Health check counters on the Gateway resource now report correct information after a gateway leader re-election occurs.
 
+**Note**: this version also fixes issues with v0.16.4, v0.16.5, v0.16.6.
```

Comment:
Most of the releases fix issues with earlier releases :wink:. How about making it clear that this replaces the previous releases:

```suggestion
**Note**: this version replaces v0.16.4, v0.16.5, v0.16.6.
```

This explains why those releases don’t have entries. Hopefully it will also make it clearer that some of the items listed were addressed in one of the replaced releases — I know I sometimes look up a change in the release notes, and look at commits in the corresponding release to find the implementation, and that wouldn’t work here if I were to look for a change in 0.16.7 that was addressed in 0.16.4. Typically in release branches it wouldn’t matter much because there aren’t all that many commits, but this series of releases in particular involved quite a lot of commits so it’s more of a problem!